README.md

```diff
@@ -41,9 +41,6 @@ This package takes advantage of non-candidate equilibrium monte carlo moves (NCM
 The heart of the package is found in `blues/ncmc_switching.py`. This holds the framework for NCMC. Particularly, it contains the integrator class that calculates the work done during the NCMC move. It also controls the lambda scaling of parameters. Currently the alchemy package is used to generate the lambda parameters for the ligand, which can potentially modify 5 parameters–sterics, electrostatics, bonds, angles, and torsions.
 The class SimNCMC in `blues/ncmc.py` serves as a wrapper for running ncmc simulations. SimNCMC.runSim() actually runs the simulation.
 
-## Example Use
```

Comment:
I'm not sure why the entire block got removed. It might have gotten accidently deleted when I was trying to resolve conflicts. It should have just been changed to reference the `blues/example.py` location.